projects/admin/src/app/record/formly/type/entity-typeahead/add-entity-local/add-entity-local.component.html

```diff
@@ -0,0 +1,22 @@
+<!--
+  RERO ILS UI
+  Copyright (C) 2019-2023 RERO
+  Copyright (C) 2019-2023 UCLouvain
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Affero General Public License as published by
+  the Free Software Foundation, version 3 of the License.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<button class="btn btn-sm btn-primary mt-2" *ngIf="canAdd" (click)="addEntity()">
+  <i class="fa fa-plus-square-o" aria-hidden="true"></i>
+  {{ 'Add local entity' | translate }}
+</button>
```

Comment:
I'm always confused about so small component (just a button). Is that could not be include into the `entity-typeahead` component ? At least, as the HTML template is very small, it could be include into the `.component.ts`

Reply:
I put the html code in the `composent.ts` file and moved the file to the typeahead root.